core/src/main/java/com/linecorp/armeria/client/ClientRequestContext.java

```diff
@@ -161,6 +165,34 @@ static ClientRequestContextBuilder builder(RpcRequest request, URI uri) {
         return new ClientRequestContextBuilder(request, uri);
     }
 
+    /**
+     * Returns the {@link ServiceRequestContext} whose {@link Service} invokes the {@link Client}
+     * {@link Request} which created this {@link ClientRequestContext}.
```

Comment:
`, or {@code null} if this client request was not made in the context of a server request.`

Reply:
Thanks!